### PR TITLE
Proxy caching and cors

### PR DESCRIPTION
**Pull request by samasastudio**
- added open CORS headers
- removed redis and middleware

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,20 +8,6 @@ services:
     working_dir: /Users/samjohnson/Documents/hrfiles/petco/description_directions_attributes_
     ports:
       - "80:3002"
-    links:
-      - "redis"
-    command: bash -c "wait-for-it.sh redis:6379 && npm install && npm run pro"
```
Comment (WDHudson) [inline docker-compose.yml:13]: Why did you remove the wait-for-it? I ended up not using it.

File: server.js [hunk 2]
```diff
@@ -2,14 +2,14 @@
 const morgan = require('morgan');
 const express = require('express');
 const path = require('path');
-const redis = require('redis');
+// const redis = require('redis');
```
Comment (WDHudson) [inline server.js:5]: Why did you get rid of Redis?

**Review by WDHudson — APPROVED** [review]
Looks good. Just curious about your thought process.

File: docker-compose.yml (hunk 1 above)
Comment (samasastudio) [reply docker-compose.yml:13]: i don't have to wait for redis to connect because i'm not using redis on my service instances anymore

File: server.js (hunk 2 above)
Comment (samasastudio) [reply server.js:5]: because i'm caching on my proxy server rather than my service.  it allows to respond with cache sooner with less API calls and avoid layered caching that can lead to data inconsistency.
